packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -321,24 +316,19 @@ export const MafsGraph = (props: MafsGraphProps) => {
                                     )
                                 }
                                 {/* Locked figures layer nested in SVG to lock to graph bounds*/}
-                                {props.lockedFigures &&
-                                    props.lockedFigures.length > 0 && (
-                                        <svg {...nestedSVGAttributes}>
-                                            <GraphLockedLayer
-                                                lockedFigures={
-                                                    props.lockedFigures
-                                                }
-                                                range={state.range}
-                                            />
-                                        </svg>
-                                    )}
+                                {props.lockedFigures.length > 0 && (
+                                    <svg {...nestedSVGAttributes}>
+                                        <GraphLockedLayer
+                                            lockedFigures={props.lockedFigures}
+                                            range={state.range}
+                                        />
+                                    </svg>
+                                )}
                             </Mafs>
                         </View>
-                        {props.lockedFigures && (
-                            <GraphLockedLabelsLayer
-                                lockedFigures={props.lockedFigures}
-                            />
-                        )}
+                        <GraphLockedLabelsLayer
+                            lockedFigures={props.lockedFigures}
+                        />
```

Comment:
So before when `lockedFigures` was optional, if it was falsy in some way, then this `GraphLockedLabelsLayer` element wouldn't be included. Now it's always included. Could that cause any side effects with interaction or the UI?

Reply:
I don't think this would render in the DOM if `lockedFigures` length is 0, because the outermost layer in `packages/perseus/src/widgets/interactive-graphs/graph-locked-labels-layer.tsx` is a React fragment. It only renders the actual labels themselves if there are any.

So I think this is fine.